src/components/containers/branding.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactNode } from 'react'
+import { isShowBranding } from 'common/utility'
+
+type BrandingProps = {
+    children: ReactNode
+}
+
+const Branding = ({ children }: BrandingProps) => isShowBranding() && <>{children}</>
```

Comment:
can we move the `isShowBranding` to DerivStore context? 
the reason is, right now every time we're using the `Branding` component, all the logic inside `isShowBranding` function will be run again. 
but if we move it to `DerivStore` we just have to run the function once and the actual value from `Context` in `Branding` component.
let me know what you think, thanks :pray:

Reply:
If we are supposed to use `IsShowBranding` a lot then let's move it to `DerivStore`.

Reply:
okay got it